.travis.yml

```diff
@@ -20,6 +20,7 @@ before_install:
   - conda create -q -n test-environment -c conda-forge python=$PYTHON_VERSION jupyter_server==0.0.5 jupyterlab_pygments==0.1.0 pytest==3.10.1 nbconvert=5.5 pytest-cov nodejs flake8 ipywidgets
   - source activate test-environment
 install:
+  - pip install https://github.com/maartenbreddels/voila-gridstack/archive/refactor_templates.zip
```

Comment:
Do we really need this? I'm working on a PR for removing gridstack, can we find another way around?